src/SFA.DAS.Payments.Monitoring.Jobs.JobService.Interfaces/IJobService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Services.Remoting;
+using Microsoft.ServiceFabric.Services.Remoting.FabricTransport;
+using SFA.DAS.Payments.Monitoring.Jobs.Messages.Commands;
+using SFA.DAS.Payments.Monitoring.Jobs.Model;
+
+[assembly: FabricTransportServiceRemotingProvider(RemotingListenerVersion = RemotingListenerVersion.V2_1, RemotingClientVersion = RemotingClientVersion.V2_1)]
+namespace SFA.DAS.Payments.Monitoring.Jobs.JobService.Interfaces
+{
+    public interface IJobService: IService
```

Comment:
Is this interface still required?

Reply:
Have removed and replaced the reference in JobService to IService and cut out the middleman